packages/connectivity/src/scp-cf/destination/destination-accessor-failure-cases.spec.ts

```diff
@@ -100,7 +108,11 @@ describe('Failure cases', () => {
       await getDestination({
         destinationName,
         jwt: subscriberServiceToken,
-        enableCircuitBreaker: false,
+        resilience: {
+          id: 'getDestination',
+          timeout: defaultResilienceOptions.timeout,
```

Comment:
[q] I guess the timeout should also be a function.

Reply:
That's right. But the `defaultResilienceOptions.timeout` is already a function.